src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -237,6 +237,7 @@ t8_forest_vtk_get_element_nodes (t8_forest_t forest, t8_locidx_t ltreeid,
  * the data related to the element (not element_data). 
  * 
  */
+/* *INDENT-OFF* */
```

Comment:
Do we need this indent off with the newly added datatypes?

Reply:
Yes, otherwise gnuindent changes it to an ugly mess.

Reply:
kk